root/static/scripts/edit/URLCleanup.js

```diff
@@ -1818,6 +1818,11 @@ const CLEANUPS = {
   'secondhandsongs': {
     match: [new RegExp('^(https?://)?([^/]+\\.)?secondhandsongs\\.com/', 'i')],
     type: LINK_TYPES.secondhandsongs,
+    clean: function (url) {
+      url = url.replace(/^(https?:\/\/)?([^/]+\.)?secondhandsongs\.com/, 'https://secondhandsongs.com');
```

Comment:
Capturing groups `(…)` in this regular expression are not used in the replacement string, thus these should be replaced with non-capturing groups `(?:…)`.